docs/plugins/mbsubmit.rst

```diff
@@ -9,11 +9,24 @@ that is parseable by MusicBrainz's `track parser`_. The prompt choices are:
 - Print the tracks to stdout in a format suitable for MusicBrainz's `track
   parser`_.
 
+- Create a new release on MusicBrainz, opens
+  https://musicbrainz.org/release/add in a new browser window with
+  fields pre-populated using existing metadata.
+
 - Open the program `Picard`_ with the unmatched folder as an input, allowing
   you to start submitting the unmatched release to MusicBrainz with many input
   fields already filled in, thanks to Picard reading the preexisting tags of
   the files.
 
+To create new releases on MusicBrainz with this plugin you need to install the
+`PyJWT`_ library with:
+
+.. code-block:: console
+
+   $ pip install beets[mbsubmit]
```

Comment:
With the switch to poetry, this is now outdated. Should be changed to the poetry equivalent.

Reply:
Is it?

https://github.com/beetbox/beets/blob/master/docs/plugins/fetchart.rst?plain=1#L12

I can add quotation marks for consistency.